### PR TITLE
updated

### PR DESCRIPTION
**Pull request by SenthilKumarSivakami**

### CODE REVIEW

File: faqs/seed-faq-general.md [hunk 1]
```diff
@@ -138,6 +138,8 @@ audit -e
 
 **Reason**: Most likely, this indicates that we detected some issues with your device configuration for SEED. For example, your Microsoft Defender could be unhealthy. As such issues pose a security risk, we revoke your access to SEED resources.
 
+However, there could be other reasons such as Microsoft Defender definitions are obsolete. If you are a mac user, such issues will self-heal and your access will be auto-restored and you will receive an email for this. In such cases, no action required from you for this.
```
Comment (cricketer94) [inline faqs/seed-faq-general.md:141]: I would phrase this more generally.

"In some situations, the issues might be rectified automatically and you might be able to regain access without any manual intervention, and you will receive the successfully onboarded email indicating your access has been restored"

**Review by cricketer94 — CHANGES_REQUESTED** [review]

File: faqs/seed-faq-general.md (hunk 1 above)
Comment (SenthilKumarSivakami) [reply faqs/seed-faq-general.md]: Fixed.

File: faqs/seed-faq-general.md [hunk 2]
```diff
@@ -140,15 +140,17 @@ audit -e
 
 In this situation, the **action required** by you may vary based on your need:
 
-- **Restore my access to SEED-protected resources:**  Raise a [service request][service-request] and specify that your SEED access revoked due to device misconfiguration. This would allow us to process the ticket accordingly. Once your device configuration is remediated, we will send you the successfully onboarded email again, indicating you can now access SEED-protected resources again.
+- **Automatically restore the access**: In some situations, the issue could be rectified automatically. In such cases, you may be able to regain access without any manual intervention and you will receive the successfully onboarded email indicating your access has been restored.
```
Comment (cricketer94) [inline faqs/seed-faq-general.md:143]: This phrasing is slightly awkward. Can we change it to 
```** Access is automatically restored, no action required**```

Comment (SenthilKumarSivakami) [reply faqs/seed-faq-general.md:143]: hmm, how will the user know that their access is automatically restored? 

so need to include info about the email.

**Access automatically restored**: No action required. Your access is automatically restored if you shortly receive an email that you are successfully onboarded to SEED.

Comment (cricketer94) [reply faqs/seed-faq-general.md]: Oh I meant just change the title. The description can remain. Sorry my bad.

Comment (SenthilKumarSivakami) [reply faqs/seed-faq-general.md]: updated. np!

**Pull request merged**